.github/run_test.sh

```diff
@@ -21,7 +23,7 @@ tests_ethtool_ring_nm.yml
 tests_ipv6_disabled_nm.yml
 tests_ipv6_nm.yml
 tests_vlan_mtu_nm.yml
-tests_wireless_nm.yml
+tests_switch_provider.yml
```

Comment:
Not sure why this change is here, I missed it before. I assume this is because the wireless test does not work on EL7. Would be good to track that this test needs to be added back to be running for EL8. This will probably need some additional support for different test lists for different environments.

Reply:
Nice, catch. I didn't notice it before. Let me check if it works on EL7. If it does not work on EL7 I will add a comment explaining it. I think it should be simple to add a support for different test lists depending on the environment.

Reply:
I have added it back. I will check the CI.

Reply:
test_wireless does not work on el7. I have added a FIXME comment.